low-level/sspl-ll.spec

```diff
@@ -133,10 +110,6 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
 # [ -f /etc/rsyslog.d/1-ssplfwd.conf ] ||
 #    cp /opt/seagate/%{product_family}/sspl/low-level/files/etc/rsyslog.d/1-ssplfwd.conf /etc/rsyslog.d/1-ssplfwd.conf
 
-# Copy init script
-[ -f $SSPL_DIR/sspl_init ] ||
```

Comment:
Just make sure all these omitted steps are covered in relevant mini provisioner stage, including this one

Reply:
We don't have any use on sspl_init with main branch. 
keeping this as low priority at this time. I will add this once again get confirmed.

Reply:
rt, we can ignore sspl_init script as its unused from long now. Infact plan to remove it in coming days.